packages/yourloops/components/language-select.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2021, Diabeloop

```

Comment:
I would have add a small text below the Copyright line, to describe the purpose of this file